src/components/CalendarMonth.jsx

```diff
@@ -104,7 +107,7 @@ export default class CalendarMonth extends React.Component {
     } = this.props;
 
     const { weeks } = this.state;
-    const monthTitle = month.format(monthFormat);
+    const monthTitle = renderMonth ? renderMonth(month) : month.format(monthFormat);
```

Comment:
So what is your suggestion?
I do this part as a code that now exist in `CalendarDay` component:
https://github.com/airbnb/react-dates/blob/master/src/components/CalendarDay.jsx#L131